.changelog/19415.txt

```diff
@@ -0,0 +1,11 @@
+```release-notes:enhancement
+resource/aws_wafv2_web_acl: Add `custom_request_handling` to `allow` and `count` actions.
+```
+
+```release-notes:enhancement
+resource/aws_wafv2_web_acl: Add `custom_response` to `block` actions.
+```
+
+```release-notes:enhancement
+resource/aws_wafv2_rule_group: Included the above changes to `rule` `allow`, `count`, and `block` actions.
+```
```

Comment:
Since the changelog generation is automated, the entries here will fall in alphabetical order, so would be best to make individual entries for the rule group resource

Reply:
Makes sense. I've pushed a fix to make them two line items.